spec/features/movies/show_spec.rb

```diff
@@ -0,0 +1,109 @@
+require "rails_helper"
+
+RSpec.describe "Movie Show Page" do 
+  it "shows the movie's title, creation year, and genre" do
+
+    paramount = Studio.create!({name: "Paramount Pictures", location: "Hollywood"})
+    jaws = paramount.movies.create({title: "Jaws", creation_year: "1976", genre: "Action"})
+
+    visit "/movies/#{jaws.id}"
+
+    expect(page).to have_content("Title: Jaws")
+    expect(page).to have_content("Year: 1976")
+    expect(page).to have_content("Genre: Action")
+  end
+
+  it "shows each movie's actors sorted from youngest to oldest" do 
+
+    paramount = Studio.create!({name: "Paramount Pictures", location: "Hollywood"})
+    jaws = paramount.movies.create({title: "Jaws", creation_year: "1976", genre: "Action"})
+    
+    roy = Actor.create!({name: "Roy Scheider", age: 36})
+    richard = Actor.create!({name: "Richard Dreyfuss", age: 28})
+    robert = Actor.create!({name: "Robert Shaw", age: 61})
+
+    jaws_actor_1 = MovieActor.create!({actor_id: roy.id, movie_id: jaws.id})
+    jaws_actor_2 = MovieActor.create!({actor_id: richard.id, movie_id: jaws.id})
+    jaws_actor_3 = MovieActor.create!({actor_id: robert.id, movie_id: jaws.id})
+
+    visit "/movies/#{jaws.id}"
+
+    expect(page).to have_content("Cast:")
+    expect(page).to have_content("Richard Dreyfuss, age: 28")
```

Comment:
you can add the orderly gem here to test that Richard Dreyfuss appears before Roy Schneider, etc